lib/spell/message.ex

```diff
@@ -218,13 +218,17 @@ defmodule Spell.Message do
 
   @doc """
   Return a new WAMP id.
+
+  To ensure the uniqueness of the new id we use :crypto.rand_bytes to generate
+  a random seed
+
+  TODO: improve `:random.uniform` using a Mersenne Twister PRNG algorithm
   """
   @spec new_id :: integer
   def new_id do
-    # TODO: hack to ensure the random generator is seeded -- will result in
-    # clock skew
-    :random.seed(:erlang.now())
-    (:math.pow(2, 53) |> round |> :random.uniform) - 1
+    << a :: 32, b :: 32, c :: 32 >> = :crypto.rand_bytes(12)
+    :random.seed(a,b,c)
+    :math.pow(2, 53) |> round |> :random.uniform
```

Comment:
I am not sure why we were decrease by 1 the random generated ID. Probably I am missing something - On that concern I wanted to ask to @jtmoulia if this implementation is correct.